tests/components/axis/test_config_flow.py

```diff
@@ -73,9 +69,10 @@ async def test_flow_manual_configuration(hass):
     }
 
 
-async def test_manual_configuration_update_configuration(hass, config_entry):
+async def test_manual_configuration_update_configuration(
+    hass, config_entry, setup_config_entry, mock_vapix_requests
+):
     """Test that config flow fails on already configured device."""
-    await setup_axis_integration(hass, config_entry)
     device = hass.data[AXIS_DOMAIN][config_entry.entry_id]
```

Comment:
Side note: We should not access hass.data in tests of integrations. It's a detail. Assert the config entry data or state instead.

Reply:
Fixed in https://github.com/home-assistant/core/pull/86644